### ISSUE TITLE
[BUG] Repository path issues on Windows

### ISSUE THREAD

**Issue by OmarFourati** [jvalue/jayvee#494]
## Steps to reproduce
1. Work with repository on windows

## Description
- Expected: works
- Actual: breaks because colon not valid filename char. on windows.

```
error: invalid path 'libs/extensions/tabular/exec/test/assets/cell-range-selector-executor/test-A1:C16.xlsx'
error: invalid path 'libs/extensions/tabular/exec/test/assets/cell-range-selector-executor/test-A1:C16.xlsx.license'
error: invalid path 'libs/extensions/tabular/exec/test/assets/cell-range-selector-executor/test-B1:C2.xlsx'
error: invalid path 'libs/extensions/tabular/exec/test/assets/cell-range-selector-executor/test-B1:C2.xlsx.license'
error: invalid path 'libs/extensions/tabular/exec/test/assets/cell-range-selector-executor/valid-A1:A4.jv'
error: invalid path 'libs/extensions/tabular/exec/test/assets/cell-range-selector-executor/valid-A1:C*.jv'
error: invalid path 'libs/extensions/tabular/exec/test/assets/cell-range-selector-executor/valid-A1:E4.jv'
error: invalid path 'libs/extensions/tabular/exec/test/assets/cell-writer-executor/test-A1:C16.xlsx'
error: invalid path 'libs/extensions/tabular/exec/test/assets/cell-writer-executor/test-A1:C16.xlsx.license'
error: invalid path 'libs/extensions/tabular/exec/test/assets/column-deleter-executor/test-A1:B2.xlsx'
error: invalid path 'libs/extensions/tabular/exec/test/assets/column-deleter-executor/test-A1:B2.xlsx.license'
error: invalid path 'libs/extensions/tabular/exec/test/assets/column-deleter-executor/test-A1:C16.xlsx'
error: invalid path 'libs/extensions/tabular/exec/test/assets/column-deleter-executor/test-A1:C16.xlsx.license'
error: invalid path 'libs/extensions/tabular/exec/test/assets/row-deleter-executor/test-A1:B2.xlsx'
error: invalid path 'libs/extensions/tabular/exec/test/assets/row-deleter-executor/test-A1:B2.xlsx.license'
error: invalid path 'libs/extensions/tabular/exec/test/assets/row-deleter-executor/test-A1:C16.xlsx'
error: invalid path 'libs/extensions/tabular/exec/test/assets/row-deleter-executor/test-A1:C16.xlsx.license'
error: invalid path 'libs/extensions/tabular/exec/test/assets/sheet-picker-executor/test-A1:C16.xlsx'
error: invalid path 'libs/extensions/tabular/exec/test/assets/sheet-picker-executor/test-A1:C16.xlsx.license'
```

**Comment by OmarFourati** [comment]
Just noticed that the files aren't even to find in my repo even if I am in the main branch.
So cloning the repo on windows just leaves out the whole "test" folder of libs/extensions/tabular/exec/

**Comment by joluj** [comment]
Windows has a maximum length for a path, by default [260 characters](https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation).

Allowing longer paths is described here: https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation.

TLDR: Run `New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1 -PropertyType DWORD -Force` in a powershell with administrator rights.

**Comment by OmarFourati** [comment]
I am not sure that this is the problem. The files doesn't even get clone with the repo. 
This path for example doesn't have more than 260 characters : libs/extensions/tabular/exec/test/assets/cell-writer-executor/test-A1:C16.xlsx. right ?
I think ( and @rhazn read it somewhere ) that it is about the naming.

**Comment by joluj** [comment]
That's true, thanks for the PR. I have also created #499 to assert that no such error comes up again. It fails now (which is good since there are invalid file names).
After #498 is merged #499 hopefully runs successfully.

**Comment by joluj** [comment]
closed by #498 and #499
